localisation/00_zuwc_united_localisation.csv

```diff
@@ -1770,7 +1760,7 @@ world_northrend_howling_fjord;Howling Fjord;Fjord Hurlant;Heulender Fjord;;;;;;;
 EVTDESC_WCCOU_200_demon;Though your soul is formally immortal, you still need to worry about your heirs, who will continue your work if you do not return from the Twisting Nether. [target_new_member.GetTitledFirstName] would be the perfect candidate to lead your legions in the future.;;Obwohl Eure Seele formell unsterblich ist, m�sst Ihr Euch immer noch um Eure Erben k�mmern, die Eure Arbeit fortsetzen werden, wenn Ihr nicht aus dem wirbelnden Nether zur�ckkehrt. [target_new_member.GetTitledFirstName] w�re [target_new_member.Getderdie] perfekte Kandidat[target_new_member.GetinFemale], um Eure Legionen in der Zukunft zu f�hren.;;;;;;;;;;;x
-EVTDESC_WCCOU_200_titanforged;The world is a dangerous place and accidents can happen, perhaps you could break down in combat or for other reasons, but there is a remedy to secure your legacy. [target_new_member.GetTitledFirstName] is a simple construct, but if you shared part of your life spark with him he could carry on your legacy should something happen to you.;;Die Welt ist ein gef�hrlicher Ort und es kann zu Ungl�cksf�llen kommen. Ihr k�nntet im Kampf oder aus anderen Gr�nden zusammenbrechen, aber es gibt ein Mittel, um Euer Erbe zu sichern. [target_new_member.GetTitledFirstName] ist eine einfache Konstruktion, aber wenn Ihr einen Teil Eures Lebensfunkens mit [target_new_member.Getihmihr] teilt, k�nnte [target_new_member.Getersie] Euer Verm�chtnis weiterf�hren, sollte Euch etwas passieren.;;;;;;;;;;;x
+EVTDESC_WCCOU_200_titanforged;The world is a dangerous place and accidents can happen, perhaps you could break down in combat or for other reasons, but there is a remedy to secure your legacy. [target_new_member.GetTitledFirstName] is a simple construct, but if you shared a part of your life spark with [target_new_member.GetHerHim], [target_new_member.GetSheHe] could carry on with your legacy, should something happen to you.;;Die Welt ist ein gef�hrlicher Ort und es kann zu Ungl�cksf�llen kommen. Ihr k�nntet im Kampf oder aus anderen Gr�nden zusammenbrechen, aber es gibt ein Mittel, um Euer Erbe zu sichern. [target_new_member.GetTitledFirstName] ist eine einfache Konstruktion, aber wenn Ihr einen Teil Eures Lebensfunkens mit [target_new_member.Getihmihr] teilt, k�nnte [target_new_member.GetSheHe] Euer Verm�chtnis weiterf�hren, sollte Euch etwas passieren.;;;;;;;;;;;x
```

Comment:
Isn't carry on someone's legacy correct?

Reply:
Hmm, seems you're right here, too. Both constructs work, I thought my one would sound better... @LordSilvermane: could you choose one?

Reply:
Like you said, they both work, but "carry on _with_ your legacy" isn't exactly a common expression, when compared to the other. I'd stick with the original, myself, if only because more English speakers will be accustomed to it.

Reply:
alright, done :)